lib/widgets/action_sheet.dart

```diff
@@ -569,7 +571,8 @@ void showMessageActionSheet({required BuildContext context, required Message mes
   final showMarkAsUnreadButton = markAsUnreadSupported && isMessageRead;
 
   final optionButtons = [
-    ReactionButtons(message: message, pageContext: pageContext),
+    if (popularEmojiLoaded)
```

Comment:
I think we can have a test checking when this should not be offered, like we do with quote-and-reply button.